manage_breast_screening/notifications/tests/queries/test_failures_query.py

```diff
@@ -62,20 +62,19 @@ def test_failures_query_data_today(self):
             {"starts_at": appt_time, "nhs_number": "9990001116", "episode_type": "S"},
             {
                 "status": "failed",
-                "nhs_notify_errors": {
-                    "errors": [
-                        {"code": "CM_INVALID_NHS_NUMBER", "title": "Invalid NHS number"}
-                    ]
-                },
+                "nhs_notify_errors": [
+                    {"code": "CM_INVALID_NHS_NUMBER", "title": "Invalid NHS number"}
+                ],
             },
         )
         appt7 = self.create_message_set(
             {"starts_at": appt_time, "nhs_number": "9990001117", "episode_type": "S"},
             {
                 "status": "failed",
-                "nhs_notify_errors": {
-                    "errors": [{"code": "CM_INVALID_VALUE", "title": "Invalid value"}]
-                },
+                "nhs_notify_errors": [
+                    {"code": "CM_INVALID_VALUE", "title": "Invalid value"},
```

Comment:
do you think we're unlikely to get any of the other potential errors then? like `CM_NULL_VALUE` or `CM_MISSING_VALUE`? Or would those be errors that we'd have to dig into ourselves, and these are the errors that it's up to the BSO to fix?

Reply:
It's a good question, I suppose null value, missing value and invalid value are in the same ballpark so I'll add these too.

Reply:
I have included these in https://github.com/NHSDigital/dtos-manage-breast-screening/pull/502/commits/8e94cc38068db171d64e15df399d2595f8c2b799